addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.udf_ui",
+    "name": "User Defined Functions Formula Editor",
+    "description": "A Seeq add-on tool for managing user-defined formula functions.",
+    "version": "1.0.0",
```

Comment:
Version `1.0.0` is already in the gallery. You'll need to rev the version number so Add-on Manager can know to display that an update is available.

Reply:
I'll look into this! Plot Curve needs the same change